api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -50,7 +54,7 @@ def _open_soma(locator: CensusLocator, context: Optional[soma.options.SOMATileDB
 
 def open_soma(
     *,
-    census_version: Optional[str] = "latest",
+    census_version: Optional[str] = None,
```

Comment:
shouldn't this be
```suggestion
    census_version: Optional[str] = "stable",
```

Reply:
Since this is a private method, I think handling the default in the body makes sense.

Reply:
Gotcha

Reply:
Actually, I can and will revert this to original behavior. I had a reason for doing this as one point in the coding evolution...